dice_ml/data_interfaces/private_data_interface.py

```diff
@@ -370,7 +374,11 @@ def prepare_query_instance(self, query_instance):
 
         else:
             raise ValueError("Query instance should be a dict, a pandas dataframe, a list, or a list of dicts")
+        return test
 
+    def prepare_query_instance(self, query_instance):
```

Comment:
Could you defined this function in https://github.com/interpretml/DiCE/blob/main/dice_ml/data_interfaces/base_data_interface.py? This will be inherited in both public and private data.

Reply:
Thanks for addressing the comment in your latest commit.